packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-graphql-schema.ts

```diff
@@ -30,9 +30,12 @@ import {
   JSONUtilities,
   pathManager,
   stateManager,
+  FeatureFlags,
+  $TSAny,
+  isCI,
```

Comment:
I don't see this import used, is there a check needed for CI?

Reply:
removed